models/Program.ts

```diff
@@ -0,0 +1,23 @@
+export enum weekdays {
+    MON = "MON",
+    TUE = "TUE",
+    WED = "WED",
+    THU = "THU",
+    FRI = "FRI",
+    SAT = "SAT",
+    SUN = "SUN",
+}
+
+export type createProgramInput = {
```

Comment:
also - add type documentation because it will be likely be used by both fe/be developers

Reply:
seems like the TypeScript convention is to capitalize types: https://www.typescriptlang.org/docs/handbook/2/objects.html